morphocut/image.py

```diff
@@ -16,7 +16,14 @@
 @ReturnOutputs
 @Output("mask")
 class ThresholdConst(Node):
-    """Set the mask of image
+    """
+    Set the mask of image
+
+    Args:
+        image (RawOrVariable or Variable[RawOrVariable]): An image file for which the mask is to be set.
```

Comment:
image (np.ndarray or Variable[np.ndarray]): Image for which the mask is to be calculated.